### ISSUE TITLE
Provided CEF installer doesnt install all files (my opinion)

### ISSUE THREAD

**Issue by Yexeed** [MCMrARM/mcpelauncher-linux#301]
At the end of install cef there is an error that prevent main.cpp to include cef_app.h:
`cp: указанная цель 'стол/mcpelauncher-linux/libs/cef/include' не является каталогом`
(on english this means that 'стол/mcpelauncher-linux/libs/cef/include' is not catalog)
I think, that the folder libs/cef/include/ is empty because this error happens. Please fix this, and i just try to copy needed files from provided archive (cef.tar.bz)

**Comment by Yexeed** [comment]
Yeah, copying contains of folder "/cef_binary_3.3112.1653.gf69054f_linux32_minimal/include/" from archive really helped :) 
![image](https://user-images.githubusercontent.com/32373441/35195874-e7de43c4-feda-11e7-9107-c2a44accedf0.png)

**Comment by coolkidsrule101** [comment]
lol

**Comment by MCMrARM** [comment]
Can't reproduce here, can you post the full output of setup_cef.sh?

**Comment by Yexeed** [comment]
oh, i find the solution just at know. Since my system is russian, my desktop folder named by "Рабочий стол" (2 words). Installer doesnt work in this cases i think.
